app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala

```diff
@@ -77,10 +82,12 @@ class PlatformInitiativesService @Inject()(
         initiativeName            = initiativeName,
         initiativeDescription     = initiativeDescription,
         currentProgress           = repos
+          .filter   (repositories => team.fold(true)(repositories.teamNames == Seq(_)))
```

Comment:
should these checks be `team.fold(true)(repositories.teamNames.contains)`? Or are we filtering for exclusively owned? If so, we should add a comment to say this is intentional.

Reply:
This was intentional, so added comments to explain this.